### DISCUSSION TITLE
Matmat's Trophy list

### DISCUSSION THREAD

**Discussion by IvanNascimento** [janstehno/cotw-companion#74] [Ideas]
Would be nice if there a screen for the collections you have. Like the trophy list

# Example
## Bison Bonanza
- European Bison
  Brown 💎 129
- European Bison
  Dark Brown 🔶 116
## David & Goliath
- European Bison
  Light Brown 🔶 112
- Canada Goose
  Brown Hybrid 🔶 7.7

**Comment by Rancher17** [comment]
I would like to know which multimounts an animal can be used in from the animal’s description. For example, if I look at the description for red fox, it will list that it can be used in the Fox’s Lunch and Foxy Foes mounts.
